main/zoe/api/zoe-helpers.md

```diff
@@ -162,12 +126,13 @@ and the offer that we are trying out has the handle `tryHandle`.
 If the `keepOffer` is no longer active, we reject the `tryOffer` with
 the `keepHandleInactiveMsg`. 
 
-If `canTradeWith` returns false for the two offers, we reject the
+If `satisfies` returns false for the two offers, we reject the
 `tryOffer`.
 
-If `canTradeWith` is true, we reallocate with Zoe by swapping the
+If `satisfies` is true, we reallocate with Zoe by swapping the
 amounts for the two offers, then we complete both offers so that the
-users will receive their payout.
+users will receive their payout. Any surplus in the swap remains with the 
+original offer (`keepHandle`).
```

Comment:
```suggestion
offer who gave the surplus.
```

I think the inclusion of `keepHandle` is coming from a misunderstanding. Both offers keep their surplus if there is any. Basically, if they give more than the other side wants, they keep the remainder.